contracts/v2/interfaces/ITokenWrappedBridgeUpgradeable.sol

```diff
@@ -8,4 +7,7 @@ interface ITokenWrappedBridgeUpgradeable {
         string memory symbol,
         uint8 __decimals
     ) external;
+
+    function mint(address to, uint256 value) external;
+    function burn(address account, uint256 value) external;
```

Comment:
Can we add here all the functions related to permit and standrad erc20 as well ?
As we mention on [here](https://github.com/0xPolygonHermez/zkevm-contracts/pull/443#issuecomment-2850230791)